helm/templates/custom-configmap.yaml

```
@@ -0,0 +1,19 @@
+{{- if .Values.customChainsConfig.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.customChainsConfig.name }}
+  namespace: {{ default .Release.Namespace .Values.namespaceOverride | quote }}
+  labels:
+    {{- include "cvms.labels" . | nindent 4 }}
+    {{- with .Values.customChainsConfig.exteraLabels }}
+      {{- toYaml . | nindent 4 }}
```

yamllint (1.35.1)

[warning] wrong indentation: expected 2 but found 6

(indentation)